Helper/PaymentStatusLabel.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Paynow\PaymentGateway\Helper;
+
+/**
+ * Class PaymentStatusLabel
+ *
+ * @package Paynow\PaymentGateway\Helper
+ */
+class PaymentStatusLabel
+{
+    public static $PENDING = 'processing';
```

Comment:
const ?

Reply:
z constami z jakiegoś powodu nie działało to:
`'payment_status_label' => __(PaymentStatusLabel::${$status})`